layout/_partials/head/head.swig

```diff
@@ -53,12 +53,8 @@
   <meta name="baidu-site-verification" content="{{ theme.baidu_site_verification }}"/>
 {% endif %}
 
-{% if theme.qihu_site_verification %}
-  <meta name="360-site-verification" content="{{ theme.qihu_site_verification }}"/>
-{% endif %}
-
 {% if theme.fancybox %}
-  {% set fancybox_css_uri = url_for(theme.vendors._internal + '/fancybox/source/jquery.fancybox.css?v=2.1.5') %}
+  {% set fancybox_css_uri = url_for(theme.vendors._internal + '/fancybox/source/jquery.fancybox.css') %}
```

Comment:
Version need for auto-updating cache.

For example:
1. If FB will `v3.5.6` and after update & redeploying will `v3.5.7` and
2. If CSS of this versions have difference and
3. If on server-side will defined long-life cache
    Then `v3.5.7` will be loaded as old `v3.5.6` with some interface bugs. Yes, developers know what need simply refresh browser's cache, but simple users (your blog visitors) don't know it.

I suggest define latest FB v3. In our case for now it `v3.5.7`. And even if we want using old v2.1.5, there will no update for this version and all will be working fine.

Reply:
fancybox2 is no longer maintained, and we didn't update fancybox3 in the past whole year (even it's updated from 3.2.5 to 3.5.7, but no one from NexT updated it). So I believe removing the version number will not affect anything.

Reply:
Soon was 3.5.6 version. Then u update to 3.5.7. Tomorrow can be 3.5.8, etc. I believe version number must present here for maximum round all possible bugs in the future – this is argument. Your arguments?

Reply:
But the truth is that the cycle of updating our plugins is significantly longer than the usual cache time of servers and browsers... However, if you can find a beautiful solution to deal with this problem (version numbers for both fancybox2 and 3), I will support it.

Another problem is that the users may not update the `theme-next-fancybox` in time, this will also cause version number issues...

Reply:
> Another problem is that the users may not update the theme-next-fancybox in time, this will also cause version number issues...

Well ,this is argument, agree. So, if u already know solution about cache refresh without modifying current file, awaiting for it.